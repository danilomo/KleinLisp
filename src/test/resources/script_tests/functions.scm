(define (square x) (* x x))
(println (square 4))
(println (square 7))

(define (add-three a b c) (+ a b c))
(println (add-three 1 2 3))