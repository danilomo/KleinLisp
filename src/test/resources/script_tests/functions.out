16
49
6